src/clap.rs

```diff
@@ -338,6 +338,12 @@ use CPU (3) as the default instead.
         .help("The amount in ms changed upon zooming.")
         .long_help("The amount of time in milliseconds changed when zooming in/out. The minimum is 1s (1000), and defaults to 15s (15000).");
 
+    let title = Arg::new("title")
+        .long("title")
+        .takes_value(true)
+        .value_name("Title")
+        .help("Sets the title of the current terminal.");
+
```

Comment:
I thought we were just doing the hostname idea and putting the custom title part aside for now. If so, change this to match the config option, and not take a value.